src/ReactNativeZoomableView.tsx

```diff
@@ -65,10 +69,17 @@ class ReactNativeZoomableView extends Component<
     contentHeight: undefined,
     panBoundaryPadding: 0,
     visualTouchFeedbackEnabled: true,
+    staticPinPosition: undefined,
```

Comment:
do all `pin` prefixes in other variable names refer to the static pin. In that case, maybe we should just call these `pinX` instead of `staticPinX`?

Reply:
I want the `static` name on this to be consistent – feels like there could be other kinds of pins than a static one